functions/_fzf_search_history.fish

```diff
@@ -20,7 +24,7 @@ function _fzf_search_history --description "Search command history. Replace the
             $fzf_history_opts |
         string split0 |
         # remove timestamps from commands selected
-        string replace --regex '^\d\d-\d\d \d\d:\d\d:\d\d │ ' ''
+        string replace --regex '^.* │ ' ''
```

Comment:
Is this greedy? Cause if so and the command itself contains a |, then it'll consume too much

Reply:
No, I tested this with commands that had │and it worked correctly. Only way this can go wrong is if the user puts │ in their `fzf_history_time_format`, but I considered it a pretty far fetched edge case so didn't handle it.

If you prefer, I could add a check to make sure there is no │ in the user's `fzf_history_time_format`.

Alternatively, if you want to feel a bit safer about the regex, I can change the `*` to `[^│]`.

Reply:
But I did just realize that I didn't handle it correctly for the preview window, so fixed that.

Reply:
> If you prefer, I could add a check to make sure there is no │ in the user's fzf_history_time_format.

Let's just put that in the docs. Can you take a stab at updating the readme? And I'll polish it for you.

> No, I tested this with commands that had │and it worked correctly.

Great, but now I'm confused why it works...?

Reply:
Ok actually you are right, my bad, the regex is greedy. I will fix it. And yes I will also update the readme.

Reply:
Ok, fixed the code and updated the README. Go ahead and polish it :)